hl7/4.2/modules/ROOT/edits/hl7-connector-config-topics.adoc

```diff
@@ -0,0 +1,269 @@
+
+The following is an example HL7 message structure schema for the
+standard v2.5.1 ADT_A05 message structure:
+
+[source,yaml,linenums]
+----
+form: HL7
+version: '2.5.1'
+imports: [ '/hl7/v2_5_1/basedefs.esl' ]
+id: 'ADT_A05'
+name: 'ADT_A05'
+data:
+- { idRef: 'MSH', position: '01', usage: M }
+- { idRef: 'SFT', position: '02', usage: O, count: '>1' }
+- { idRef: 'EVN', position: '03', usage: M }
+- { idRef: 'PID', position: '04', usage: M }
+- { idRef: 'PD1', position: '05', usage: O }
+- { idRef: 'ROL', position: '06', usage: O, count: '>1' }
+- { idRef: 'NK1', position: '07', usage: O, count: '>1' }
+- { idRef: 'PV1', position: '08', usage: M }
+- { idRef: 'PV2', position: '09', usage: O }
+- { idRef: 'ROL', position: '10', usage: O, count: '>1' }
+- { idRef: 'DB1', position: '11', usage: O, count: '>1' }
+- { idRef: 'OBX', position: '12', usage: O, count: '>1' }
+- { idRef: 'AL1', position: '13', usage: O, count: '>1' }
+- { idRef: 'DG1', position: '14', usage: O, count: '>1' }
+- { idRef: 'DRG', position: '15', usage: O }
+- groupId: 'PROCEDURE'
+  count: '>1'
+  usage: O
+  items:
+  - { idRef: 'PR1', position: '17', usage: M }
+  - { idRef: 'ROL', position: '18', usage: O, count: '>1' }
+- { idRef: 'GT1', position: '20', usage: O, count: '>1' }
+- groupId: 'INSURANCE'
+  count: '>1'
+  usage: O
+  items:
+  - { idRef: 'IN1', position: '22', usage: M }
+  - { idRef: 'IN2', position: '23', usage: O }
+  - { idRef: 'IN3', position: '24', usage: O, count: '>1' }
+  - { idRef: 'ROL', position: '25', usage: O, count: '>1' }
+- { idRef: 'ACC', position: '27', usage: O }
+- { idRef: 'UB1', position: '28', usage: O }
+- { idRef: 'UB2', position: '29', usage: O }
+----
+
+This shows the list of segments comprising the `ADT_A05` message structure
+including the segment groups PROCEDURE and INSURANCE. Since this example is
+the standard definition (not the lax version), it includes mandatory
+segments (indicated with `usage: M`) as well as optional segments
+(indicated with `usage: O`). The full set of usage codes used for HL7 are:
+
+* `C` for Conditional (equivalent to Optional)
+* `M` for Mandatory
+* `O` for Optional
+* `U` for Unused (accepted without warning when reading, but not present
+in the data passed on from the read operation and ignored when writing.)
+
+The possible number of occurrences of a segment or group is defined by the
+count value. This defaults to a value of `1`.
+
+To delete segments from the message structure or change segment requirements from mandatory to optional (or vice versa), make the change in your copy of the schema and use the modified version in your application.
+
+You can add segments that are not present in the original message
+structure definition. First remove the position values from all existing segment and group definition lines in the schema, otherwise you need to renumber everything following an added segment.
```

Comment:
The use of "First" here led me to expect a sequence of actions (task?), but no chronological actions follow?

Reply:
```suggestion
structure definition. First, remove the position values from all existing segment and group definition lines in the schema; otherwise, you must renumber everything following an added segment.
```